### ISSUE TITLE
Kafka-Rest Output Plugin not starting with Kafka Output Plugin Errors

### ISSUE THREAD

**Issue by tanermelikoglu** [fluent/fluent-bit#1248]
## Bug Report
I use the image image: fluent/fluent-bit:1.0.0
I want to deploy kafka-rest plugin via Kubernetes with following configmap:

```
apiVersion: v1
kind: ConfigMap
metadata:
  name: fluent-bit-config
  namespace: elk
  labels:
    k8s-app: fluent-bit
data:

  fluent-bit.conf: |
    [SERVICE]
        Flush         1
        Log_Level     info
        Daemon        off
        Parsers_File  parsers.conf
        HTTP_Server   On
        HTTP_Listen   0.0.0.0
        HTTP_Port     2020

    @INCLUDE input-kubernetes.conf
    @INCLUDE filter-kubernetes.conf
    @INCLUDE output-kafka-rest-proxy

  filter-kubernetes.conf: |
    [FILTER]
        Name                kubernetes
        Match               kube.*
        Kube_URL            https://kubernetes.default.svc.cluster.local:443
        Merge_Log           On
        K8S-Logging.Parser  On

  input-kubernetes.conf: |
    [INPUT]
        Name              tail
        Tag               kube.*
        Path              /var/log/containers/*.log
        Parser            docker
        DB                /var/log/flb_kube.db
        Mem_Buf_Limit     5MB
        Skip_Long_Lines   On
        Refresh_Interval  10

  output-kafka-rest-proxy: |
    [OUTPUT]
        Name            kafka-rest
        Match           *
        Host            https://my-rest-proxy-url
        Topic           logs-topic
        Port            80
        tls             on
        tls.verify      off
        tls.ca_file    ca-file
        tls.crt_file    crt-file
        tls.key_file    key-file

```

but i get following **kafka output plugin** errors;

```
[2019/04/04 13:21:28] [error] [config] kafka.0 don't support TLS
[2019/04/04 13:21:28] [ info] [storage] initializing...
[2019/04/04 13:21:28] [ info] [storage] in-memory
[2019/04/04 13:21:28] [ info] [storage] normal synchronization mode, checksum disabled
[2019/04/04 13:21:28] [ info] [engine] started (pid=1)
[2019/04/04 13:21:28] [error] [out_kafka] config: no brokers defined
[2019/04/04 13:21:28] [error] [out_kafka] failed to initialize
[2019/04/04 13:21:28] [error] [output] Failed to initialize 'kafka' plugin
```

**Comment by singhjagmohan1000** [comment]
I think there is something wrong with the configuration. As in output plugin at `Host` mention the host without https:// and http://. Try this.

**Comment by tanermelikoglu** [comment]
@singhjagmohan1000 i tried it via http out but the same error stack.

**Comment by singhjagmohan1000** [comment]
@tanermelikoglu try this `Host            my-rest-proxy-url`. No https:// or http://.

**Comment by edsiper** [comment]
looks like the core its loading kafka instead of kafka-rest plugin. Would you please try version 1.0.6 and see if the issue still happens ?

**Comment by jlloh** [comment]
Running into the same issue. Seems to occur after version 0.12

```
jl@ubuntu-vm:/etc/td-agent-bit$ sudo docker run -ti fluent/fluent-bit:0.12 /fluent-bit/bin/fluent-bit -i cpu -o kafka-rest -p host=xxxx -p=8082 -p topic=fluent-bit-test -m '*'
Fluent-Bit v0.12.19
Copyright (C) Treasure Data

[2019/04/17 05:04:45] [ info] [engine] started
```

```
jl@ubuntu-vm:/etc/td-agent-bit$ sudo docker run -ti fluent/fluent-bit:0.13 /fluent-bit/bin/fluent-bit -i cpu -o kafka-rest -p host=xxxx -p=8082 -p topic=fluent-bit-test -m '*'
Fluent-Bit v0.13.8
Copyright (C) Treasure Data

[2019/04/17 05:05:22] [ info] [engine] started (pid=1)
[2019/04/17 05:05:22] [error] [out_kafka] config: no brokers defined
[2019/04/17 05:05:22] [error] [out_kafka] failed to initialize
```

**Comment by edsiper** [comment]
Thanks for report the issue, there were two causes, both are fixed now on git master:

- 8261f632
- ae4863aa

**Comment by jaxonpickett** [comment]
Is this fix available in docker yet?

**Comment by edsiper** [comment]
FIx will be available with 1.1 release (next week).

**Comment by tanermelikoglu** [comment]
working on 1.1. Thx @edsiper
